roles/chameleon_mariadb/tasks/deployment.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Add mariadb backup job
+  include_role:
+    name: chameleon.periodic_task
+  vars:
+    task_name: "mariadb_backup_{{ mariadb_backup_timer_suffix }}"
+    task_command: "sh {{ deployment_dir }}/cc-ansible --site {{ deployment_site_config_dir }} --tags mariadb_backup"
+    task_calendar: "23:00"
```

Comment:
`daily`?

Reply:
I set the "backup" script to run at 23:00, and the "copy to central location" script to 00:00. I'd like to specify the actual time here to make sure one runs before the other.